### PR TITLE
add labels to section headers in each sidebar

### PR DESCRIPTION
**Pull request by sw-yx**
this is an attempt to solve this problem

image

### CODE REVIEW

**Comment by sw-yx** [comment]
the search doesnt change until this gets merged and... as far as i can tell... this change has no outward visible changes. imo we should give this a try and see how the search reranks

File: sidebars.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 module.exports = {
   serverSidebar: [
-    'server/introduction',
+    {
+      type: 'doc',
+      label: 'Temporal Server', 
+      id: 'server/introduction',
+    },
```
Comment (flossypurse) [inline sidebars.js:7]: I don't think this will have any effect.
These are defined in the file already.

**Comment by flossypurse** [comment]
I think this will have the effect we are looking for:
https://github.com/temporalio/documentation/pull/383

**Comment by sw-yx** [comment]
yeah i think so. lets try your thing first!

**Pull request closed without merging**